webhook/webhook.go

```diff
@@ -245,12 +256,18 @@ func (wh *Webhook) Run(stop <-chan struct{}) error {
 		QuietPeriod: wh.Options.GracePeriod,
 	}
 
+	nextProto := map[string]func(*http.Server, *tls.Conn, http.Handler){}
```

Comment:
can we add a comment that
// If TLSNextProto is not nil, HTTP/2 support is not enabled automatically.